siliconcompiler/scheduler.py

```diff
@@ -62,17 +60,34 @@ def _deferstep(chip, step, index, status):
         if 'slurm_account' in chip.status:
             username = chip.status['slurm_account']
             schedule_cmd.extend(['--acount', username])
+        # Only delay the starting time if the 'defer' Schema option is specified.
+        defer_time = self.get('option', 'scheduler', 'defer')
+        if defer_time:
+            schedule_cmd.extend(['--begin', defer_time])
     elif scheduler_type == 'lsf':
         # TODO: LSF support is untested and currently unsupported.
         schedule_cmd = ['lsrun']
 
+    # User-defined pre/post-processing steps for the Slurm script.
+    # Unlike most other scheduler parameters, we should keep these pre/post-process blocks in the
+    # ephemeral status dictionary, so that they cannot persist across remote connections.
+    # The best place to set these values would be a custom target created by the cluster administrator.
+    script_prefix, script_suffix = '\n', '\n'
```

Comment:
Per offline discussion, replace this with "if scheduler script does not already exist" logic.